JSX.md

```diff
@@ -58,19 +54,25 @@ The `as` operator is available in both `.ts` and `.tsx` files.
 
 # Type Checking
 
-In order to understand type checking with JSX you must first understand intrinsic elements vs value-based elements. Given a JSX expression `<expr />`, is `expr` referring to something intrinsic to the environment (ie, a div or span in a DOM environment) or to a custom component that you've made? This is important for two reasons:
+In order to understand type checking with JSX you must first understand the difference between intrinsic elements value-based elements.
+Given a JSX expression `<expr />`, `expr` may either refer to something intrinsic to the environment (e.g. a `div` or `span` in a DOM environment) or to a custom component that you've created.
+This is important for two reasons:
 
-1. For React, intrinsic elements are emitted as strings, like `React.createElement('div')`, whereas a component you've created is not: `React.createElement(MyComponent)`.
+1. For React, intrinsic elements are emitted as strings (`React.createElement("div")`), whereas a component you've created is not (`React.createElement(MyComponent)`).
 2. The types of the attributes being passed in the JSX element should be looked up differently. Intrinsic element attributes should be known *intrinsically* whereas components will likely want to specify their own set of attributes.
```

Comment:
It means that for something like `<abbr title="foo">bar</abbr>` the "title" attribute should "just be known" by the type system whereas for something like...

```
import MyComponent = require('./myComponent');

<MyComponent specialValue="foobar" />
```

... it's really the responsibility of MyComponent to specify what attributes it accepts and what their types are.